projects/typescript-examples/contracts/ScratchStorage/contract.e2e.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { Config } from '@algorandfoundation/algokit-utils'
+import { registerDebugEventHandlers } from '@algorandfoundation/algokit-utils-debug'
+import { algorandFixture } from '@algorandfoundation/algokit-utils/testing'
+import { Address } from 'algosdk'
```

Comment:
If possible, do not use `algosdk`. If this is inevitable right now, we should raise this to the makerX team. I believe we can use the `Account` type in utils and get the `addr` string field instead.

Reply:
I don't think `Account` is re-exported, unless you have an example? So may need to keep this for now.

Reply:
nvm I misunderstood, I'll push a fix.

Reply:
fixed in [04f8b54](https://github.com/algorandfoundation/devportal-code-examples/pull/28/commits/04f8b54ea7c161cfd35bb6ce1c9727eb0cadd782) for both PRs